core/domain/user_domain.py

```diff
@@ -373,3 +373,79 @@ def validate(self):
             raise utils.ValidationError(
                 'Expected can_review_questions to be a boolean value, '
                 'found: %s' % type(self.can_review_questions))
+
+
+class ModifiableUserData(python_utils.OBJECT):
+    """Domain object to represent change in allowed attributes for UserSettings
```

Comment:
.. to represent the new values in a UserSettingsModel change submitted by the Android client.

Reply:
Done.